include/coap3/coap_dtls.h

```diff
@@ -86,7 +88,7 @@ coap_tls_version_t *coap_get_tls_library_version(void);
  * @return @c 1 if successful, else @c 0.
  */
 typedef int (*coap_dtls_security_setup_t)(void* tls_session,
-                                        struct coap_dtls_pki_t *setup_data);
+                                          struct coap_dtls_pki_t *setup_data);
```

Comment:
This `struct` now should be obsolete.

Reply:
Oops - looks like I started on this one and got distracted.  Code update pushed.